pkg/apis/k0s/v1beta1/kuberouter.go

```diff
@@ -36,6 +36,9 @@ type KubeRouter struct {
 	PeerRouterASNs string `json:"peerRouterASNs"`
 	// Comma-separated list of global peer ASNs
 	PeerRouterIPs string `json:"peerRouterIPs"`
+	// ExtraArgs are extra arguments to pass to kube-router
+	// Can be also used to override the default k0s managed kube-router arguments
```

Comment:
```suggestion
	// ExtraArgs are extra arguments to pass to kube-router.
	// Can be also used to override the default k0s managed kube-router arguments.
```

Reply:
hmm, none of the existing attributes use period

Reply:
The comment will be copied over to the OpenAPI spec, in which it will loose any line breaks. Adding periods at the end of the sentences makes it more readable there.

But sure, it's just a nit.

Reply:
I'd rather keep this consistent with other fields and maybe do a chore PR later to fix all in one go

Reply:
Alright.

Reply:
https://github.com/k0sproject/k0s/issues/3924